src/App.js

```diff
@@ -1,25 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Home from "./pages/Home.jsx";
```

Comment:
```suggestion
import Home from "./pages/Home";
```

You don't need to identify the extension.